### ISSUE TITLE
Inline option for custom linters throws "Invalid inline option"

### ISSUE THREAD

**Issue by hunterxyz** [lesshint/lesshint#430]
I'm using the version 4.1.1 and I'm using it through the API

I adde my .lesshintrc file my custom linter
```json
{
...
"linters": [
        "no-font-size.js"
    ]
}
```

```less
.hero-new-style {
    font-size: 52px; // lesshint no-font-size: false
}
```
It should import the linter rules if they are included into the .lesshintrc file

**Comment by jwilsson** [comment]
Hi!
I think this is due to the hyphens in your rule name. It seems like it only accepts alphanumeric characters (where it should obviously accept anything).

I'll push a fix shortly.

**Comment by jwilsson** [comment]
Fixed in `4.1.2`.
